test/integration/account-creation-tls-test.js

```diff
@@ -18,7 +18,7 @@
 //     sslCert: path.join(__dirname, '../keys/cert.pem'),
 //     auth: 'tls',
 //     webid: true,
-//     idp: true,
+//     multiuser: true,
```

Comment:
This test file is completely commented out...

Reply:
I know; I'm not sure about the purpose of this file, I just search & replaced even in the commented code in case it is ever taken up again.